x-pack/plugins/ml/public/application/settings/calendars/list/header.tsx

```diff
@@ -0,0 +1,137 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/*
+ * React component for the header section of the calendars list page.
+ */
+
+import type { FC } from 'react';
+import React from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+
+import {
+  EuiSpacer,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiLink,
+  EuiText,
+  EuiTextColor,
+  EuiButtonEmpty,
+} from '@elastic/eui';
+
+import { MlPageHeader } from '../../../components/page_header';
+import { useMlKibana } from '../../../contexts/kibana/kibana_context';
+
+interface Props {
+  isDst: boolean;
+  totalCount: number;
+  refreshCalendars: () => void;
+}
+
+export const CalendarsListHeader: FC<Props> = ({ totalCount, refreshCalendars, isDst }) => {
+  const {
+    services: {
+      docLinks: { links },
+    },
+  } = useMlKibana();
+  const docsUrl = links.ml.calendars;
+  return (
+    <>
+      <MlPageHeader>
+        {isDst ? (
+          <FormattedMessage
+            id="xpack.ml.settings.calendars.listHeader.calendarsTitle"
+            defaultMessage="Daylight savings calendars"
```

Comment:
Nit, for consistency I guess `Daylight savings time calendars` is better here?

Reply:
Updated in https://github.com/elastic/kibana/pull/193605/commits/db72ef0440d4e606a284040d561af0f981647fc5